tests/test_environment.py

```diff
@@ -11,6 +11,17 @@ def test_runs_locally(temporary_installed_project):
     run_flow(flow_name(temporary_installed_project))
 
 
+def test_duff_project_runs_locally(temporary_duffproject):
+    """Flow should run but `myduffproject` won't have been installed since the project is duff.
+    The flow itself should test whether the error handling is working (fixes bug
+    https://github.com/nestauk/metaflow_extensions/issues/23)
+    """
+    with ch_dir(temporary_duffproject / "myduffproject"):
+        run_flow(
+            temporary_duffproject / "myduffproject/myduffproject/flows/duff_flow.py"
+        )
+
```

Comment:
```suggestion
    assert b"project environment did not find a root file" in proc.stdout
    
    
```

Reply:
Note that the `click.secho` message goes to `stdout`.
I think this is fine? If you disagree can pass `err=True` to `click.secho` to print to `stderr`.

Reply:
It did cross my mind whether or not this should constitute an error, but I decided that the current format of effectively being a "warning" would suffice, although it does make debugging more cryptic. If anything, I only found this because of the bug. 

Also IMO at the stage where we divert to `stderr` we should also raise an error; and so, counter question: should we *raise* an error here? I'm leaning to yes, which I could implement in this PR and ask for re-review

Reply:
actually, I think the quasi warning status is fine, since the perks of having `project_environment` will break in production, prompting the developer to check the logs anyway. I'll merge